node/conversions/conversionlimit.go

```diff
@@ -0,0 +1,220 @@
+package conversions
```

Comment:
I remeber writing this code in `pegnet` library https://github.com/pegnet/pegnet/blob/master/modules/conversions/conversionlimit.go
why it's here?

Reply:
We will remove `conversionlimit.go` from `pegnet` repository as conversion is used in pegnetd only. @sigrlami

Reply:
@dragonheaven that's a breaking change for 3rd party miners who are using `pegnet` as a dependency